tests/phpunit/includes/abstract-testcase.php

```diff
@@ -1532,6 +1532,23 @@ public function _make_attachment( $upload, $parent_post_id = 0 ) {
 		return $id;
 	}
 
+	/**
+	 * Uploads given file and creates an attachment post from it.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param array $filename       Absolute path to the file to upload.
+	 * @param int   $parent_post_id Optional. Parent post ID.
+	 *
+	 * @return int|WP_Error The attachment ID on success. The value 0 or WP_Error on failure.
```

Comment:
In which case we get `0` return value? Per `_make_attachment` return type is should return `WP_Error on failure.

```suggestion
	 * @return int|WP_Error The attachment ID on success, WP_Error object on failure.
```

Reply:
Good catch! Originally I copied the docs from `_make_attachment`. However, it changed since I created the PR. It's now changed to match `_make_attachment`.